components/src/dynamo/common/lora/manager.py

```
@@ -112,3 +119,38 @@ def is_cached(self, lora_uri: str) -> bool:
+def get_lora_manager() -> Optional[LoRAManager]:
+    """Get the LoRAManager singleton, initializing it on first call if enabled."""
+    existing = _lora_manager.get()
+    if existing is not None:
+        return existing
+
+    if os.environ.get("DYN_LORA_ENABLED", "").lower() not in ("true", "1", "yes"):
+        return None
+
+    try:
+        return _lora_manager.get_or_init(_init_lora_manager)
+    except Exception as e:
```

Ruff (0.15.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)